wandb/internal/internal.py

```diff
@@ -247,14 +247,14 @@ def wandb_internal(  # noqa: C901
     logger.info("W&B internal server running at pid: %s", pid)
 
     system_stats = None
-    if not settings._disable_stats:
+    if not settings._disable_stats and not settings.offline:
```

Comment:
In the old cli I believe we still collected system metrics in dryrun mode.

Reply:
Good point, that was a mistake.  I will keep stats enabled.